### PR TITLE
Added note about --blank flag

### PR DESCRIPTION
**Pull request by ajaykarwal**
Added instructions about installing Jekyll with a blank slate using the --blank flag.

### CODE REVIEW

File: docs/_docs/quickstart.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ If you encounter any unexpected errors during the above, please refer to the alr
 * To install the Jekyll site into the directory you're currently in, run `jekyll new .` If the existing directory isn't empty, you can pass the `--force` option with `jekyll new . --force`.
 * `jekyll new` automatically initiates `bundle install` to install the dependencies required. (If you don't want Bundler to install the gems, use `jekyll new myblog --skip-bundle`.)
 * By default, the Jekyll site installed by `jekyll new` uses a gem-based theme called [Minima](https://github.com/jekyll/minima). With [gem-based themes](../themes), some of the directories and files are stored in the theme-gem, hidden from your immediate view.
+* We recommend setting up Jekyll with a gem-based theme but if you want to start with a blank slate, use `jekyll new myblog --blank`
```
Comment (DirtyF) [inline docs/_docs/quickstart.md]: 👍  for mentioning the option. I'd rather stay neutral on gem-based theme though:

If you prefer to start from a blank state then use the `--blank`option: `jekylll new --blank myblog`.

Comment (ajaykarwal) [inline docs/_docs/quickstart.md]: This was added based on the suggestion I received on twitter - https://twitter.com/jekyllrb/status/822593046550298626

Comment (DirtyF) [inline docs/_docs/quickstart.md]: My bad, didn't see this.

Comment (ajaykarwal) [inline docs/_docs/quickstart.md]: No probs, So is this being merged and pushed to the website?

**Comment by DirtyF** [comment]
@jekyllbot: merge +doc

File: docs/_docs/quickstart.md (hunk 1 above)
Comment (DirtyF) [inline docs/_docs/quickstart.md]: 🎉

**Pull request merged**